tests/unit/authors/test_authors_receivers.py

```diff
@@ -63,6 +73,93 @@ def test_name_variations():
             'Richard Ellis'])
 
 
+@httpretty.activate
+def test_phonetic_block_generation_ascii():
+    from flask import current_app
```

Comment:
Seems like the `app` argument was provided by `mock` decorator. After removing `mock`, the method was not receiving any argument.